auth/customers.go

```diff
@@ -18,7 +35,8 @@ func IsCustomer(repo types.Repository) (bool, error) {
 	var err error
 	var found bool
 	c := http.Client{}
-	request, _ := http.NewRequest(http.MethodGet, "https://raw.githubusercontent.com/alexellis/derek/master/.CUSTOMERS", nil)
+	customersURL := findCustomersURL()
+	request, _ := http.NewRequest(http.MethodGet, customersURL, nil)
 	res, doErr := c.Do(request)
 	if err != nil {
```

Comment:
@alexellis is this an error?  I was expecting `doErr` to be evaluated in the `if`?

Reply:
Yes that's a bug waiting to happen. Can you fix?